pallets/liquidity-pools/src/message.rs

```diff
@@ -30,6 +34,8 @@ pub const TOKEN_NAME_SIZE: usize = 128;
 // The fixed size for the array representing a tranche token symbol
 pub const TOKEN_SYMBOL_SIZE: usize = 32;
 
+pub const MAX_BATCH_MESSAGES: u32 = 256;
```

Comment:
256 messages or bytes? I think for messages, that is too much ^^

Reply:
16 seems more reasonable?